awx/api/views/__init__.py

```diff
@@ -1683,6 +1683,34 @@ def get_queryset(self):
         return qs
 
 
+class BulkView(APIView):
+    _ignore_model_permissions = True
+    permission_classes = [IsAuthenticated]
+    allowed_methods = ['GET']
+
+    def get(self, request, format=None):
+        '''List top level resources'''
+        data = OrderedDict()
+        data['bulk_host_create'] = reverse('api:bulk_host_create', request=request)
+        return Response(data)
+
+
+class BulkHostCreateView(APIView):
+    _ignore_model_permissions = True
+    permission_classes = [IsAuthenticated]
+    allowed_methods = ['GET', 'POST']
```

Comment:
I think you should set `serializer` so that we can get meaningful data in OPTIONS, b/c that still applies. If you POST, there are certain fields that need to be provided, and those should follow the standard structure.

Reply:
Got the OPTIONS method to show up now by adding:

```
    renderer_classes = [
        renderers.BrowsableAPIRenderer,
        JSONRenderer,
    ]
```
to the view, and this finds the right template for the "description"...but still not getting the `actions` stuff to show up right re: what you need for POST

Reply:
@jbradberry helped me figure this out, I needed to inherit from GenericAPIView